addons/account_edi/data/cron.xml

```diff
@@ -3,9 +3,9 @@
         <field name="name">EDI : Perform web services operations</field>
         <field name="model_id" ref="model_account_edi_document"/>
         <field name="state">code</field>
-        <field name="code">model.search([('state', 'in', ('to_send', 'to_cancel'))])._process_documents_web_services(job_count=20)</field>
+        <field name="code">model._cron_process_documents_web_services(job_count=20)</field>
         <field name="interval_number">1</field>
-        <field name="interval_type">hours</field>
+        <field name="interval_type">days</field>
```

Comment:
It still is, we don't have enough data yet (and we won't before Odoo v16) to fully trust triggers and completely disable the recurrent cron.

Reply:
But then the cron triggering triggers will not be trustworthy either?

Reply:
@Julien00859 You really make me put everything in question this way.  Because if triggers are only deleted when the job is run, then this is just a safety fallback, but if there some indeterministic way in which triggers are deleted, you really scare me, because then our retrigger of the cron is prone to be deleted as well and an invoice could be in the loop indefinitely, at least theoretically.